docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -16,7 +16,10 @@ which stores the information required to interface with the external system.
 
 {es-sec} uses these external APIs to send cases:
 
-* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]
+* {sn}: https://developer.servicenow.com/dev.do#!/reference/api/rome/rest/c_ImportSetAPI[Import Set API]
++
+IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later and must be <<post-upgrade-deprecated-sn-connector, updated>> to ensure you have access to new connector enhancements. For example, you can push incident updates from cases that are using connectors created in version 7.15.0 or earlier. However, pushing incident updates from rules is a newer enhancement and you must update your connector or create a new one to use it. 
```

Comment:
```suggestion
IMPORTANT: {sn} ITSM and SecOps connectors created in {stack} version 7.15.0 or earlier use the https://developer.servicenow.com/dev.do#!/reference/api/quebec/rest/c_TableAPI[Table API]. They are marked as deprecated after you upgrade to version 7.16.0 or later and must be <<post-upgrade-deprecated-sn-connector, updated>> to ensure you have access to new connector enhancements. For example, you can push incident updates from cases using connectors created in version 7.15.0 or earlier. However, pushing incident updates from rules in {stack} 7.16.0 and later requires you to update your connector or create a new one to use it. 
```

Reply:
Agree with the first edit, but am leaning towards keeping the last sentence as it is because I think it flows better with the one that comes before it.